modules/ROOT/pages/authentication-in-http-requests.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: http, authentication, security, users, connectors, anypoint, studio, oauth, basic auth, digest
 
-Use the authentication that you configure in HTTP requests when your Mule app is sending requests to a service that requires authentication, such as the Github OAuth2 server described in <<OAuth2 - Authorization Code>>. In this case, your Mule app is the client. On the other hand, when you want to protect your API or app from _receiving requests_ from unauthorized access, use an API Manager policy, such as the link:/api-manager/external-oauth-2.0-token-validation-policy[OAuth 2.0 Access Token Enforcement Using External Provider] policy.
+Use the authentication that you configure in HTTP requests when your Mule app is sending requests to a service that requires authentication, such as the Github OAuth2 server described in <<OAuth2 - Authorization Code>>. In this case, your Mule app is the client. On the other hand, when you want to protect your API or app from _receiving requests_ from unauthorized access, use an API Manager policy, such as the xref:api-manager::external-oauth-2.0-token-validation-policy.adoc[OAuth 2.0 Access Token Enforcement Using External Provider] policy.
```

Comment:
I think this one should be 
> xref:2.x@api-manager::external-oauth-2.0-token-validation-policy.adoc

Reply:
That is, in fact, where it leads. By not putting a version, it chooses the canonical version, which in this case is 2.x. If you think the version should be locked, then we would want to specify it.